GoogleForCreators/web-stories-wp
# Library: Remove Animations Pane

## Summary
Original ticket was to make library pane tabs scrollable with animation and page layouts coming in. Verified that the animation tab/pane was legacy and not needed anymore so no need for the scroll & just removed animation pane code & experiment.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
NA

## Testing Instructions
open story editor -> verify no animation tab
open experiments -> verify that experiment to enable animation tab in library is gone

---



Fixes #5691
